README.md

```diff
@@ -1,2 +1,233 @@
+```xml
+<Button x:Name="logoutButton" Width="Auto" Foreground="White" Background="Black" FontWeight="Bold" 
+        FontSize="14" Click="Logout_button_click" Margin="944,10,9.6,650.2">
+    <StackPanel Orientation="Horizontal">
+        <Image Source="Assets/logout-white.png" Width="20" Height="20"/>
+    </StackPanel>
+</Button>
+```
+
+Clicking on the logout link would trigger the SLO flow.
+
+## Integrating OIDC SDK to your existing .NET application
+This section will guide you on integrating OIDC into your existing .NET application with the Asgardio Dotnet OIDC SDK.
+This allows a .NET application (i.e. Service Provider) to connect with an IDP using OpenID Connect protocol.
+This guide consist with the following sections.
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Installing the SDK](#installing-the-sdk)
+  * [Using Nuget Package Manager](#using-nuget-package-manager)
+  * [Using the library DLL](#using-the-library-dll)
+- [Login](#login)
+- [Logout](#logout)
+- [Get User Info](#get-user-info)
+
+### Introduction
+These instructions will guide you on integrating OIDC into your .NET application with the Asgardio Dotnet OIDC SDK. This allows an application (i.e. Service Provider) to connect with an IDP using OIDC protocol.
+
+### Prerequisites
+1. Microsoft Windows 8 (Or server equivalent) or greater.
+2. .NET Framework Standard 4.5.x or greater.
+3. Visual Studio 2017 Community or greater.
+
+### Installing the SDK
+#### Using Nuget Package Manager
+1. Open the Nuget Package Manger.
+2. Search for `io.asgardio.dotnet.oidc.sdk`.
+3. Include it with the suggested required dependencies for the project/solution. 
+
+Alternatively, you can also run the following command in the Package Manager CLI as shown below. 
+
+``Install-Package io.asgardio.dotnet.oidc.sdk -Version 1.0.0``
+
+#### Using the library DLL
+To build the project from the source, follow the instructions given below.
+
+1. Download the `io.asgardio.dotnet.oidc.sdk.dll` file from [here](link).
+2. Add the `io.asgardio.dotnet.oidc.sdk.dll` file as a Reference in your Visual Studio project.
+3. Build the project. 
+
+Once you have installed the SDK, create a file named `App.config` as shown below and place it in the application path.
+
+```xml
+<configuration>
+    <appSettings>
+        <add key="ClientId" value="fXZNpFBa3aNOCQ6rKU8ldsxT_WAa" />
+        <add key="ClientSecret" value="6x2fQgrT_Ov2R2OMbzfJ5yQjvVEa" />
+        <add key="AuthorizationEndpoint" value="https://localhost:9443/oauth2/authorize" />
+        <add key="TokenEndpoint" value="https://localhost:9443/oauth2/token" />
+        <add key="UserInfoEndpoint" value="https://localhost:9443/oauth2/userinfo" />
+        <add key="LogoutEndpoint" value="https://localhost:9443/oidc/logout" />
+        <add key="RedirectURI" value="http://localhost:8080/callback/" />
+        <add key="PostLogoutRedirectURI" value="http://localhost:8080/postlogout/" />
```

Comment:
Concern with the app context path as mentioned above

Reply:
Resolved in https://github.com/asgardio/asgardio-dotnet-oidc-sdk/pull/4/commits/3945c817476dfe36be1ba0eac68d777f7d6dd006